src/app.rs

```diff
@@ -359,7 +359,7 @@ pub trait Hooks: Send {
     ///
     /// # Errors
     /// If fails returns an error
-    fn init_logger(_config: &config::Config, _env: &Environment) -> Result<bool> {
+    fn init_logger(_ctx: &AppContext, _env: &Environment) -> Result<bool> {
```

Comment:
I believe the `_env: &Environment` parameter is redundant.

Since `Environment` is already included as part of `AppContext` (accessible via `ctx.environment`), it's better to pass only the `AppContext` and remove the separate `Environment` parameter.

Reply:
Good point, done